app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Api
+  module V1
+    class UsersController < Api::V1::ApiController
+      def update
+        current_user.update!(user_params)
+      end
+
+      def render_update_success
+        render json: { user: current_user }
```

Comment:
This is ok, but I think it would be better to have a view (JBuilder view) to render this kind of stuff, it's better since you can:
1. Have one place to render users
2. Customize response

Reply:
Great, @JuanfraM talked to me about this, gonna check it out! Tnx!